lib/shared/app/utils/parse-version.js

```diff
@@ -1,3 +1,3 @@
 // https://github.com/ef4/ember-browserify#the-workaround
 import Semver from 'npm:semver'; // eslint-disable-line no-unused-vars
-export { default } from 'shared/utils/parse-version';
+export { satisfies, parse, comparePart, compare, minorVersion } from 'shared/utils/parse-version';
```

Comment:
If `lib/shared/addon/components/new-catalog/component.js` is updtated to import from `shared/utils/parse-version` can't this file go away entirely?

Reply:
yes it could but any engines that may need to consume the api wouldn't be able to find it.